components/pango_display/src/handler_image.cpp

```diff
@@ -431,7 +543,7 @@ void ImageViewHandler::Special(View& view, pangolin::InputSpecial inType, float
         const float d[2] = {p1,p2};
         const float is[2] = {rview.x.Size(),rview.y.Size() };
         const float df[2] = {is[0]*d[0]/(float)view.v.w, is[1]*d[1]/(float)view.v.h};
-        ScrollView(-df[0], -df[1]);
```

Comment:
This looks like it changes scroll direction behaviour on MacOS? Was it behaving incorrectly for you?

Reply:
I actually encountered this recently when trying to build a GUI on top of master pangolin, it looks like you drive-by reversed this in a recent commit: https://github.com/stevenlovegrove/Pangolin/commit/69b9d866579a7e292f60be0b6eec73699896757a

With this change, panning e.g. ImageViews has vertical motion flipped. I suspect this is an OS issue and the change you made "fixed" it for Mac, but broke it for Windows/Linux.

Reply:
I see - the problem you see must be for right-click panning on win / linux? The SpecialInput method is generally only used on macos, but there is a reference to it https://github.com/stevenlovegrove/Pangolin/blob/837842ff93c0cafaa264071522a4e8ce16780524/components/pango_display/src/handler_image.cpp#L523 where it may be called on other platforms. I imagine reversing it there instead should make everything work correctly for all platforms.

Reply:
@stevenlovegrove How do you want debug this, should we fix the behavior in the Special function you pointed us or just locally on the pointed line 434?

Reply:
Let me merge this now so you're no longer completely blocked, and then if you were able to try my suggested fix to ensure it works on Linux / Win that would be great. If not, I can try to do it later.

Reply:
Thanks @stevenlovegrove! Much appreciated